app/models/item.rb

```diff
@@ -1,9 +1,3 @@
 class Item < ActiveRecord::Base
-  validates :name, presence: true, uniqueness: true
-
-  validates :description, presence: true
-
-  validates :price, presence: true
-  
-  validates :image, presence: true
+  validates :name, :description, :price, :image, presence: true, uniqueness: true
 end
```

Comment:
Okay so this is a valid edit. However, I made the point in our meeting with Ilana that it makes little sense to have the uniqueness be set to true for things like the image or price. The image's are defaulted to a url. A merchant probably has prices that are the same for multiple items as well.